src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -185,6 +185,15 @@ void AppCommandlineArgs::_buildParser()
     maximized->excludes(fullscreen);
     focus->excludes(fullscreen);
 
+    auto* windowOpt = _app.add_option("-w,--window",
+                                      _windowTarget,
+                                      RS_A(L"CmdWindowTargetArgDesc"));
+    windowOpt;
+
+    // maximized->excludes(windowOpt);

```

Comment:
Any TODO or issue associated with there?

Reply:
Oh, you know, I'm gonna yank this dead code. In the scenario where you do `wt -w 100 --maximized` and explicitly want the new window to be created with the ID 100, maximized, then this should work. So I can't have the parser force these args to be exclusive.